### PR TITLE
Error on <br> tags in source files

### PR DESCRIPTION
**Pull request by jdbaldry**
They are almost never semantically correct: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element

### CODE REVIEW

**Comment by clayton-cornell** [comment]
Tested and it works as expected.

But.... should this maybe be implemented as a **Warning** to start with? I have a feeling that it will block a LOT of PRs if it's set to error (based on a quick grep of a few of the doc repositories).  Or should we fix the repos first and then merge this PR?

**Comment by jdbaldry** [comment]
> But.... should this maybe be implemented as a Warning to start with? I have a feeling that it will block a LOT of PRs if it's set to error (based on a quick grep of a few of the doc repositories). Or should we fix the repos first and then merge this PR?

That's an important consideration but we don't actually run Vale in CI anywhere except in this repository where both warnings and errors block the builds.

**Review by clayton-cornell — APPROVED** [review]

File: vale/Grafana/Paragraphs.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+extends: script
+scope: raw
+level: error
+link: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element
+message: |
+  br elements must be used only for line breaks that are actually part of the content, as in poems or addresses.
```
Comment (jdbaldry) [inline vale/Grafana/Paragraphs.yml:6]:
```suggestion
  `br` elements must be used only for line breaks that are actually part of the content, as in poems or addresses.
  
  Outside of tables, replace the use of `br` elements with semantic Markdown.
  In tables, use semantic HTML elements like `p` for paragraphs, and `ul` or `ol` for lists.
```

**Review by jdbaldry — COMMENTED** [review]
Proposing a better message

File: vale/Grafana/Paragraphs.yml (hunk 1 above)
Comment (clayton-cornell) [reply vale/Grafana/Paragraphs.yml:6]: > In tables, use semantic HTML elements like `p` for paragraphs, and `ul` or `ol` for lists.

Should this maybe include the `<` and `>`?

Comment (jdbaldry) [reply vale/Grafana/Paragraphs.yml:6]: I'm open to it, I probably switch every time I write something.

Google seems to omit the brackets: https://developers.google.com/style/semantic-tagging
MDN includes them: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p

**Review by robbymilo — APPROVED** [review]

**Pull request merged**